data-es2016plus.js

```diff
@@ -1662,7 +1658,6 @@ exports.tests = [
           opera10_50: true,
           chrome30: true,
           node0_12: true,
-          iojs: true,
```

Comment:
`iojs: true` here indicates it was explicitly tested; it's better to have explicit tests preserved and not just inherit the data.

Reply:
Fixed